pkgs/tools/system/hw-probe/default.nix

```diff
@@ -0,0 +1,108 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeWrapper
+
+# Required
+, hwinfo
+, dmidecode
+, smartmontools
+, pciutils
+, usbutils
+, edid-decode
+
+# Recommended
+, withRecommended ? false # Install recommended tools
+, mcelog
+, hdparm
+, systemd
+, acpica-tools
+, drm_info
+, mesa-demos
+, memtester
+, sysstat
+, cpuid
+, util-linuxMinimal
+, xinput
+, libva-utils
+, inxi
+, vulkan-utils
+, i2c-tools
+, opensc
+
+# Suggested
+, withSuggested ? false # Install (most) suggested tools
+, hplip
+, sane-backends
+, pnputils ? null # pnputils (lspnp) isn't current in nixpkgs and appears to be poorly maintained
```

Comment:
Note that pnputils is not available in nixpkgs and appears to have been removed from debian I think it was - for being poorly maintained